readme.md

```diff
@@ -0,0 +1,161 @@
+# Remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+**[remark][]** is a tool that transforms markdown with plugins.
+These plugins can inspect and change your markup.
+You can use remark on the server, the client, CLIs, deno, etc.
+This plugin for Visual Studio Code lets you format and lint markdown files using
+remark.
+
+## Contents
+
+*   [When should I use this?](#when-should-i-use-this)
+*   [Use](#use)
+    *   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## When should I use this?
+
+If your project is checked using [`remark-cli`][remark-cli] and you use Visual
+Studio Code to edit markdown files, it’s recommended to use this extension.
+
+## Use
+
+To start using this for a project, [`remark-cli`][remark-cli] needs to be
+configured first.
+For example, install `remark-cli` and `remark-preset-lint-recommended`:
```

Comment:
It’s not clear why `lint-recommended` is needed. As this example isn’t just about VS Code, it might be good to explain more what this example shows